docs/reach/rs2/led-status.md

```diff
@@ -1,48 +1,31 @@
 Reach RS2 has several LEDs which are used as status indicators for different parts of the system.
 
-| System part | Indicator |
-|-----------|------|
-|<br>Power LEDs|<div style="text-align: center;"><img src="../img/reachrs2/led-status/power-leds.png" style="width: 150px;"></div>  |
-|<br>Power Button|<div style="text-align: center;"><img src="../img/reachrs2/led-status/power-button.png" style="width: 50px;"></div>  |
-|<br>Network LED|<div style="text-align: center;"><img src="../img/reachrs2/led-status/network-led.png" style="width: 50px;"></div>  |
-|<br>RTK Status LED|<div style="text-align: center;"><img src="../img/reachrs2/led-status/rtk-status-led.png" style="width: 50px;"></div>  |
+![](../img/reachrs2/led-status/rs2.png){.reciever}
+
+<div class="column-wrapper"> 
+    <div class="column-led"><div class="column-header"><h3>Battery LEDs </h3></div><div class="column-content rs2-battery"><div class="row"><img src="../img/reachrs2/led-status/booting.gif"><span>Loading</span></div>
+        <div class="row"><img src="../img/reachrs2/led-status/full-charge.jpg"><span>Full charge</span></div>
+        <div class="row"><img src="../img/reachrs2/led-status/charging.gif"><span>Charging</span></div>
+        <div class="row"><img src="../img/reachrs2/led-status/low-battery.jpg"><span>Low battery</span></div>
```

Comment:
We also have 2 more statuses:
1. Not enough charge to boot
2. Turning OFF
More info here: https://docs.emlid.com/reachrs2/led-status/